### DISCUSSION TITLE
Can this gba cartridge be reflashed?

### DISCUSSION THREAD

**Discussion by JuaninX** [sanni/cartreader#508] [General Chat]
This gba cartridge can be reflashed, it's a pokemon ruby.
![IMG_20220907_124745](https://user-images.githubusercontent.com/109401405/188935370-43a75fae-1a63-4b1b-923f-95561cb712aa.jpg)

there is no inscription, just some solder bridges.

**Comment by Androxilogin** [comment]
Lift the sticker and look up the number printed on the chip.

**Comment by JuaninX** [comment]
The chip has written: NEC D431000AGW-70LL
9630H7045

![IMG_20220908_104947](https://user-images.githubusercontent.com/109401405/189154942-9010a1cc-7718-4852-9a20-65db65201759.jpg)

**Comment by Icesythe7** [comment]
You need the number under the battery that chip is the sram (save data) the one under the battery is the nor flash chip (rom data)

**Comment by JuaninX** [comment]
This is the number under the battery
M6mgd977
W66ckt
Bg0 314206

![IMG_20220908_224017](https://user-images.githubusercontent.com/109401405/189261302-6acfca17-84d7-4bb9-a0ce-69a08b732824.jpg)

**Reply by Icesythe7** [reply]
Can't find a datasheet for that chip only https://www.omo-ic.com/chip/d76/m6mgd977-1190.html but I'd say you can query cfi with common commands and prolly figure it out pretty easily

**Comment by JuaninX** [comment]
apparently this is an old repro, thanks for the help!
